api/src/queries/survey/survey-update-queries.ts

```diff
@@ -139,6 +139,7 @@ export const putSurveyDetailsSQL = (
       lead_first_name = ${data.lead_first_name},
       lead_last_name = ${data.lead_last_name},
       location_name = ${data.location_name},
+      geojson = ${JSON.stringify(data.geometry)},
```

Comment:
do we need to stringify the json?

Reply:
yes, otherwise it isn't valid

Reply:
Does it also fail if we just wrap it in quotes?